.github/workflows/firebase-hosting-merge.yml

```diff
@@ -11,12 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm install && npm run build
+      - run: npm ci && npm run build
```

Comment:
Did you keep the default option on this one? I think this might have to stay as `npm install && npm run build`. There isn't an `npm ci` script in our package.json that I can see.

Reply:
I mean, on the other hand it seems like the hosting is working 😂

Reply:
Oh my goodness, for some reason I didn't see any notifications that anyone had reviewed 😄 so I'm just seeing your comments here. Apologies!!!

Reply:
It's good that it seems like it's working! When following along with the [Notion doc](https://www.notion.so/How-to-set-up-CI-CD-for-project-repos-3dc27b787d344218b74060d47532b0b6) @segdeha shared, in step 2, substep 10 for initializing the project, I did specify that `npm install && npm run build` should run before every deploy, so maybe `npm ci` is something build into the process? I did a brief search for a reference to it in the firebase docs, but with no luck.

Reply:
I think it will be fine this way, I just looked up the difference between npm ci and npm install [here](https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci#:~:text=npm%20ci%20(named%20after%20Continuous,it%20will%20throw%20an%20error.), so I think it should be fine. Worst case scenario, we just do the firebase hosting set-up process again.

Reply:
I think you're right @laurenmbeatty - thank you for linking to that SO. 👍